data_models/install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Run this script with 'sudo -u postgres install.sh'
+
+# This script defaults to postgres version 9.5.
+# To use a different version set the POSTGRES_VERSION env var
+# (i.e. 'sudo POSTGRES_VERSION=9.4 -u postgres install.sh')
+if [ -z $POSTGRES_VERSION ]; then POSTGRES_VERSION=9.5; fi
+
+createdb bioinformatics_mill
+
+psql bioinformatics_mill -c "CREATE ROLE bioinformatics_mill_user WITH LOGIN PASSWORD 'bioinformatics_password';"
```

Comment:
This password is only for use during development. This script won't be used in production. This doesn't actually go through Docker either, it's to be run on the host machine because that's where the database needs to be hosted from.